test/README.md

```diff
@@ -1 +0,0 @@
-Run these scripts from the repo root.
```

Comment:
If you don't mind I'd like to keep these test scripts.
Otherwise generally LGTM

Reply:
Ok, sounds good to me. :-)